### ISSUE TITLE
OnlyM skips during playback

### ISSUE THREAD

**Issue by dreemoutloud** [AntonyCorbett/OnlyM#193]
We have a problem where all video playback blips/jumps for a second at regular intervals of a couple of minutes. What could this be?

**Comment by AntonyCorbett** [comment]
@dreemoutloud what video playback mode are you using please?

**Comment by AntonyCorbett** [comment]
@dreemoutloud Sorry I should have been more specific. In Settings => Videos & Audio => Rendering engine.

**Comment by dreemoutloud** [comment]
Hi @antonycorbett

 We are running version 1.0.0.19 , I don’t think we have that option. Time for an update?

**Comment by AntonyCorbett** [comment]
@dreemoutloud ah yes :) Please update and try the "Media Foundation" mode. Let me know if you still have problems.

**Comment by LvDaniel** [comment]
@AntonyCorbett is this something we can do to avoid those delays when we're using Media Foundation? I've checked and video drivers are up to date.
When using SoundBox with MF or WMP I haven't seen those issues.
In case there isn't any workaround, we might need to use SoundBox for playing videos. This doesn't happen too often but still wold be nice if OnlyM can be fixed to be used as media player.

**Comment by AntonyCorbett** [comment]
@LvDaniel if I could reproduce the problem then I'd certainly be in a better position to fix it. I assume you have the subtitles option turned off. If you have any other ideas about when it does and when it doesn't happen then please let me know.

**Comment by LvDaniel** [comment]
@AntonyCorbett yes subtitle option is turned off but even if I turn it on I don't see any improvement. I have  the same behavior on the home PC.
I have a mp4 file which is delayed around 8 secs. Maybe you want to try with it to see if works ok for you.
Thanks for your help and support.
[Group.zip](https://github.com/AntonyCorbett/OnlyM/files/2776928/Group.zip)

**Comment by AntonyCorbett** [comment]
@LvDaniel Great - thanks for the sample video. I can reproduce the problem and it looks to be associated with the low bitrate. I should be able to fix it.

**Comment by AntonyCorbett** [comment]
@LvDaniel This is caused by a log-standing bug in WPF when rendering some H.264 codec video files on a secondary monitor. SoundBox worked around it by having the media window overlap the primary display by 1 pixel! The video should play fine on your primary monitor or if you use the "--nogpu" command-line option (of course, neither of these are practical solutions). I'll open another issue.

**Comment by LvDaniel** [comment]
@AntonyCorbett thanks for the info and for opening a new issue. If I can help you with some tests please let me know.

**Comment by LvDaniel** [comment]
Sure, I'll post on #195.
